tests/unit/components/sl-date-range-picker-test.js

```diff
@@ -95,30 +95,45 @@ test( 'Latest start date is the based on max date and end date', function( asser
     );
 });
 
-test( 'Events from start date input are removed upon willClearRender', function( assert ) {
+test( 'changeDate listener is added and removed from the correct namespace', function( assert ) {
     const component = this.subject();
-
-    this.render();
-
-    const startDateInput = this.$( '.sl-daterange-start-date input' )[ 0 ];
+    const inputElement = this.$( '.sl-daterange-start-date input' );
     const jQueryData = Ember.get( Ember.$, '_data' );
+    const eventData = Ember.get( jQueryData( inputElement[0], 'events' ), 'changeDate' );
```

Comment:
Same comment left on similar logic in sl-date-picker-test.js file applies here as well.